src/rsz/src/RepairDesign.cc

```diff
@@ -153,9 +158,25 @@ void RepairDesign::repairDesign(
   resize_count_ = 0;
   resizer_->resized_multi_output_insts_.clear();
 
+  if (gain_buffering) {
+    // If gain-based buffering is enabled, we need to override slews in order to get
+    // good required time estimates.
+    for (int i = resizer_->level_drvr_vertices_.size() - 1; i >= 0; i--) {
+      Vertex* drvr = resizer_->level_drvr_vertices_[i];
+      for (auto rf : {RiseFall::rise(), RiseFall::fall()}) {
+        sta_->setAnnotatedSlew(drvr, resizer_->tgt_slew_corner_,
```

Comment:
Any true user annotated slews will be lost.

Reply:
That’s an awkward one. I was hoping people don’t use annotations in this context. So should we annotate if there isn’t an annotation already, later revert only those annotations we added?